frontend/amundsen_application/static/js/features/Widgets/SearchBar/InlineSearchResults/SearchItemList/SearchItem/index.tsx

```diff
@@ -28,16 +28,23 @@ export interface OwnProps {
 export type SearchItemProps = StateFromProps & OwnProps;
 
 export class SearchItem extends React.Component<SearchItemProps, {}> {
+  // disable eslint prefer-destructuring
+  onItemSelect = this.props.onItemSelect.bind(this); // eslint-disable-line
```

Comment:
This is not a common thing to do.
What are the reasons that made you change it from what you had on line 33?

Reply:
I was playing around trying to find syntax that passed the eslint prefer-destructuring rule but couldn't get it to work. I can put it back to how it was (getting the callback out of props inside the `onViewAllResults` arrow function)

Reply:
Not sure, how is that?

Reply:
Actually nevermind, this way works and doesn't require having eslint ignore the line:

```
onViewAllResults = (e) => {
    logClick(e);
    const { resourceType, onItemSelect } = this.props;
    onItemSelect(resourceType, true);
  };
```